src/components/Tabs/Tabs.st.css

```diff
@@ -24,37 +30,37 @@
     IndicatorColor: --overridable;
 }
 
+:vars {
+    calculatedBorderColor: wixApply(color, wixApply(fallback, value(IndicatorColor), wixApply(opacity, wixApply(color, value(defaultIndicatorColor)), 0.2)));
+}
```

Comment:
Why separate `:var` declarations?

Reply:
Theres a convention in this project of distinguishing between variables that are overridable and those that aren't.
I've changed it a bit. Soon to be pushed.